### PR TITLE
Added csrf protection for account deactivation

### PR DESCRIPTION
**Pull request by oisinq**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -14,3 +14,7 @@ spring.mvc.view.prefix: /WEB-INF/jsp/
 spring.mvc.view.suffix: .jsp
 
 welcome.message: Hello Mkyong
```
Comment (charliekelly13) [inline src/main/resources/application.properties:16]: Whats this about?

Comment (oisinq) [reply src/main/resources/application.properties:16]: No idea

Comment (oisinq) [reply src/main/resources/application.properties:16]: I think this was always there, I can get rid of it

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -13,4 +13,6 @@ spring.jpa.hibernate.ddl-auto = update
 spring.mvc.view.prefix: /WEB-INF/jsp/
 spring.mvc.view.suffix: .jsp
 
-welcome.message: Hello Mkyong
+server.port=8081
```
Comment (DarraghClarke) [inline src/main/resources/application.properties:16]: Should this branch still have the feature to move users who connect on this port to 8443?

Comment (oisinq) [reply src/main/resources/application.properties:16]: Yes sir

Comment (oisinq) [reply src/main/resources/application.properties:16]: Oh wait no that's gone for some reason, my bad

**Pull request merged**